docs/DeveloperGuide.adoc

```diff
@@ -235,12 +235,89 @@ image::CommitActivityDiagram.png[]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
-// tag::dataencryption[]
-=== [Proposed] Data Encryption
+=== Tab Switch Feature
+Tab switch feature is a type of Command that allows users to switch to respective windows for using different features in the system. System UI structure is generally divided into 2 parts;
 
-_{Explain here how the data encryption feature will be implemented}_
+==== Implementation Structure
+
+.TabCommand Implementation Class Diagram
+image::TabCommandClassDiagram.png[]
```

Comment:
There could be a minor error in the arrow from Tab to UiPart. Navigability arrow instead of inheritance.

![InkedUML tab_LI](https://user-images.githubusercontent.com/42164491/68081457-542fd780-fe49-11e9-8f90-e962dd893a66.jpg)